ocs_ci/ocs/cluster.py

```diff
@@ -954,6 +954,28 @@ def validate_replica_data(pool_name, replica):
     raise PoolNotFound(f"Pool {pool_name} not found on cluster")
 
 
+def get_byte_used_by_pool(pool_name):
+    """
+
+    Check byte_used value for specific pool
+
+    Args:
+        pool_name (str): name of the pool to check replica
+
+    Returns:
+        integer: The amount of byte stored from pool.
+
+    """
+    ceph_df_detail_output = get_ceph_df_detail()
+    pool_list = ceph_df_detail_output.get("pools")
+    for pool in pool_list:
+        if pool.get("name") == pool_name:
+            logger.info(f"{pool_name}")
```

Comment:
Logging the pool name only doesn't tell much. I suggest logging the pool name along with the byte used.

Reply:
This is leftovers from debugging. The pool and the bytes are logged in the test itselft
```log.info(f"pool {cbp_name} has {ceph_pool_byte_used} bytes used")```
I'll delete the pool logged here